drivers/platform/aducm3029/i2c.c

```diff
@@ -73,6 +74,10 @@ static uint32_t last_bitrate = UNINITIALIZED_BITRATE;
 /** Save the current slave_address to not change it each time */
 static uint8_t last_address = UNINITIALIZED_ADDRESS;
 
+/* Data to send in prologue */
+static uint8_t *prologue_data = NULL;
```

Comment:
I would add this inside desc.

Reply:
i2c does not have an extra header. Do we want to add it just for this?

Reply:
you can declare here aducm_i2c_desc and add to i2c_desc->extra.
You don't need an extra file